packages/date-picker/src/vaadin-date-picker-overlay-content.js

```diff
@@ -233,6 +233,11 @@ class DatePickerOverlayContent extends ControllerMixin(ThemableMixin(DirMixin(Po
 
   static get properties() {
     return {
+      scrollDuration: {
```

Comment:
It is an internal component, so I thought it doesn't matter whether the property should start with `_` or not.